src/renderer.ts

```diff
@@ -248,6 +251,13 @@ export class Renderer {
           );
           t.expression(() => factory.draw());
           t.expression(() => (x += staves.get(1)!.getWidth()));
+          if (x > 1500) {
```

Comment:
Seems like the renderer needs to know about its container's context. We will need to use [ResizeObserver](https://developer.mozilla.org/en-US/docs/Web/API/ResizeObserver) and re-render on resize.